src/assets/i18n/fr.json

```diff
@@ -478,8 +478,8 @@
   "{n, plural, =0 {No Errors} one {# Error} other {# Errors}}": "",
   "{n, plural, =0 {No Tasks} one {# Task} other {# Tasks}} Configured": "",
   "{n, plural, one {# boot environment} other {# boot environments}} has been deleted.": "",
-  "{n, plural, one {# docker image} other {# docker images}} has been deleted.": "",
-  "{n, plural, one {# docker image} other {# docker images}} has been updated.": "",
+  "{n, plural, one {# docker image} other {# docker images}} has been deleted.": "{n, plural, one {# image docker} other {# images docker}} {n, plural, one {a été supprimée} other {ont été supprimées}}.",
```

Comment:
I'll merge as is, but so that you know you can also do this next time:
```
{n, plural, one {# image docker a été supprimée} other {# images docker ont été supprimées}}.
```

Reply:
Thks understood ;-)